cmake/modules/Findhidapi.cmake

```diff
@@ -62,28 +61,32 @@ find_library(hidapi_LIBRARY
 )
 mark_as_advanced(hidapi_LIBRARY)
 
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(
-  hidapi
-  DEFAULT_MSG
-  hidapi_LIBRARY
-  hidapi_INCLUDE_DIR
-)
-
 # Version detection
-if (EXISTS "${hidapi_INCLUDE_DIR}/hidapi.h")
-    file(READ "${hidapi_INCLUDE_DIR}/hidapi.h" hidapi_H_CONTENTS)
+if (EXISTS "${hidapi_INCLUDE_DIR}/hidapi/hidapi.h")
```

Comment:
We are not sure if the header is in the /hidapi sub-directory. 

I am sure the original code was working when it has been written by @uklotzde here:
https://github.com/mixxxdj/mixxx/pull/3730
 
So for a reliable version detection we need to use the 
```Cflags: -I${includedir}/hidapi```
Info.

Reply:
> We are not sure if the header is in the /hidapi sub-directory.

Yes we are. This is easily verifiable by looking up the list of files installed by packages. Here is the libhidapi-dev package from Ubuntu 18.04:
https://packages.ubuntu.com/bionic/amd64/libhidapi-dev/filelist

It installs /usr/include/hidapi/hidapi.h.

I am not sure this code ever worked before.

Reply:
I can confirm that for Ubuntu Groovy. Maybe @uklotzde can explain why the code was written like that. Is it different with fedora?

Reply:
No, it is the same on Fedora:
```
❯ rpm -ql hidapi-devel
/usr/include/hidapi
/usr/include/hidapi/hidapi.h
/usr/lib64/libhidapi-hidraw.so
/usr/lib64/libhidapi-libusb.so
/usr/lib64/pkgconfig/hidapi-hidraw.pc
/usr/lib64/pkgconfig/hidapi-libusb.pc
```

Reply:
Same on Arch:

```
$ yay -Ql hidapi | grep -F hidapi.h
hidapi /usr/include/hidapi/hidapi.h
```